### ISSUE TITLE
Relation between the differents PCB

### ISSUE THREAD

**Issue by NeodarZ** [HardlightVR/HL-electronics#1]
Hello,

first thanks for open-sourced your work. But you have only open-sourced the code,
PCBA design, electronics BOM, some tutorials and some notes but you forgotten
the documentation for connect PCB between them...

Or it's just maybe me who doesn't know how to use Autodesk EAGLE... But I don't
see anything related to cables in EAGLE projects...

Thanks in advance

**Comment by hierophect** [comment]
You are correct, this was an oversight. I'll dig up the docs for the cable assembly. In the meantime, it shouldn't be too difficult to find all of the required mating parts from the female headers listed in the BOM. 

Cables are one of the trickier parts of the assembly because the final version of the Hardlight ended up using a the Molex ClickMate series, which is hard to assemble outside a factory (it has very tiny crimps and expensive tooling). If your goal is to assemble your own Hardlight it might be beneficial to modify the PCBs to use a more hobby-assembly-friendly version like the Molex SL series (http://tech.mattmillman.com/info/crimpconnectors/). Otherwise whatever factory you work with should be able to help guide the cable assembly process.

**Comment by NeodarZ** [comment]
Hello,

ok no problem thank you. Yes there is some informations about cables in https://github.com/HardlightVR/HL-hardware/tree/master/Cables but my problematic is on relation between PCB. With some deduction I can understand who is connected to who but it's add some work for me that already be done by you. So if you found the documentation about this, it's perfect.

And thanks for the tip about cables connectors.

**Comment by hierophect** [comment]
Actually, my mistake, I hadn't realized that that documentation had already been uploaded. All cable documentation is in "Instructions" and "Cable Insertion Instructions". Is there something about these two documents you are finding unclear? The insertion instructions, in particular, has a detailed layout of how each cable in the system is connected in the suit.

**Comment by NeodarZ** [comment]
Sorry I think I didn't explain very well...

The two file you speak about give some informations about cable management and a
little on how to build the cables.

If I take an example. I want to build the cable between the controller board and
a motor board. This two card need some communication and energy to work
correctly. So for build this cable I need to know for each connector pin of the
controller board where to connect on pins of a motor board. Have you a document
who explain this ?

I hope my question is clear this time, I apologize in advance if it's not the case.

**Comment by hierophect** [comment]
Well, when the instructions are followed properly, the cables all latched (they only snap in one way) and non-polar (they'll still make the right connections even if you flip the direction of the cable). So you don't need to know what the actual purpose of each wire is when assembling them - if the cables are made correctly it's impossible to plug them in "wrong". 

But in if you want to look that up anyway, you will need to open the .brd files in Eagle or export them to Kicad or whatever PCB assembly program you are using. The schematic files will also have the connector net attachments "in order" but you'll still need to reference the board file to see the physical orientation. From there, it shouldn't be too tough to use whatever "inspect" tool your program provides to see the net name of each pin on the connector. We didn't add silkscreen names to them since there simply wasn't room, and it isn't really relevant information for the factory assembling the cables. 

If you need to learn/warm up on PCB layout Autodesk has some good tutorials. 
https://www.autodesk.com/products/eagle/blog/pcb-layout-basics-component-placement/

**Comment by NeodarZ** [comment]
Hi,

ok thanks for this informations, in the Instructions.pdf file their is only cable assembly diagram for D2M cable but not for the others but now I think I have finally understood after rereading the schematic control board.

And I think why I have been confused the first time I analyzed the project, in EAGLE project their is two PCB Board Wifi but their is no mention of this components elsewhere on Github or in schematic control board. Can you tell me more about that or redirect me to a document who tell more?

Thanks a lot for all informations you already give me.

**Comment by hierophect** [comment]
I realize the issue now. While the instructions are present, the cable diagrams were missed. I'll try to find those and upload them. 

The wifi boards were never actually manufactured and tested for quality control, only the PCB designs were completed. So we have limited sourcing information.
